composer/algorithms/progressive_resizing/progressive_resizing.py

```diff
@@ -129,6 +129,8 @@ class ProgressiveResizing(Algorithm):
                                                 mode='resize',
                                                 initial_scale=1.0,
                                                 finetune_fraction=0.2,
+                                                delay_fraction=0.2,
```

Comment:
default delay fraction should be consistent everywhere

Reply:
Default is 0.0 everywhere for delay_fraction. It's 0.2 in this example.

Reply:
ah, good point; I failed at reading the surrounding context, sorry